__tests__/app.test.js

```diff
@@ -47,4 +48,40 @@ describe("app", () => {
         });
     });
   });
+  describe("/api/reviews", () => {
+    it("200 GET - responds with and array of review objects with the correct properties", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.reviews).toBeInstanceOf(Array);
+          expect(body.reviews).toHaveLength(13);
```

Comment:
Great length assertion, super important when later "expects" are nested in a forEach 👍